src/main/java/org/takes/rq/RqMultipart.java

```diff
@@ -506,6 +491,55 @@ private static StringBuilder fakeBody(final Request... dispositions)
                 .append(String.format("--%s--", BOUNDARY));
             return builder;
         }
+
+        /**
+         * This class is using a decorator pattern for representing
+         * a one-to many header-body relation.
+         */
+        private static class RqMultipartRequest implements Request {
```

Comment:
@sebing sorry for late notice, but I've just noticed it. This class is used to create `Fake`, so please rename it to `FakeMultipartRequest`